codemp/rd-rend2/tr_shader.cpp

```diff
@@ -310,7 +310,7 @@ static int NameToSrcBlendMode( const char *name )
 	{
 		if ( r_ignoreDstAlpha->integer )
 		{
-			return GLS_DSTBLEND_ONE;
+			return GLS_SRCBLEND_ONE;
```

Comment:
It's simply wrong, found them because of Eez. The function should return a scr blend not a dst blend. This simply crashes for example the JK2 main menu.